jest/initApiPromise.js

```diff
@@ -0,0 +1,20 @@
+import {WsProvider} from '@polkadot/api';
+import Types from '../packages/types/src/injects.ts';
+import config from '../config';
+import {Api as ApiPromise} from '../packages/api/src/Api';
+
+const initApiPromise = async () => {
+  const providerUrl = config.wsProvider[`${process.env.TEST_TYPE}`] || 'ws://localhost:9944';
+  const wsProvider = new WsProvider(providerUrl);
+
+  console.log('providerUrl', providerUrl);
+  console.log('process.env.TEST_TYPE: ', process.env.TEST_TYPE);
+
+  const types = {...Types};
+  const api = await ApiPromise.create({provider: wsProvider, types});
```

Comment:
We don't need **types** while creating api instance... it will implicitly have it (ref - https://github.com/cennznet/api.js/blob/98501f94cdb2aede89ee44d34bd92ac1e46f3289/packages/api/src/Api.ts#L107)

Reply:
Tried without types, but not working, so have to inject it. But will double check later.

Reply:
Yeah, will fix it.